src/java.base/share/classes/java/util/regex/Pattern.java

```diff
@@ -755,6 +755,14 @@
  *    within a group; in the latter case, flags are restored at the end of the
  *    group just as in Perl.  </p></li>
  *
+ *    <li><p> In Perl, <i>free-spacing mode</i> (which is called <i>comments
```

Comment:
Looks good overall, one suggestion is to perhaps re-phrase the opening of the paragraph as the preceding  paragraph also starts with "In-Perl".

Reply:
Good idea. Re-worded to "Free-spacing mode in Perl (called comments..."